### PR TITLE
User gets notification when new comments are added

### PR DESCRIPTION
**Pull request by GeorgeJLewis**
https://www.pivotaltracker.com/story/show/158257202
* adds feature file with user story
* adds selenium driver with chrome
* adds happy path scenario

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -48,4 +48,21 @@
...
+  window = open_new_window
+  switch_to_window(window)
+  user = User.find_by(email: email)
+  login_as(user, scope: :user)
+end
+
+Given("He is on the show page for {string}") do |recipe_title|
+  recipe = Recipe.find_by(title: recipe_title)
+  visit recipe_path(recipe)
+
```
Comment (magnus-thor) [inline features/step_definitions/basic_steps.rb]: remove empty line

File: spec/models/recipe_spec.rb [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
-    end 
-  end
+  # describe 'Broadcast notification' do
+  #   it 'on create' do
+  #     expect{ create(:recipe)}
+  #     .to have_broadcasted_to('notifications')
+  #     .from_channel(WebNotificationsChannel)
+  #     .with({message: '<p>MyRecipe was created!</p>'})
+  #   end
+  # end
```
Comment (magnus-thor) [inline spec/models/recipe_spec.rb]: uncomment or delete commented out lines

File: features/user_gets_notification_when_new_comments_are_added.feature [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    Given We have the following user:
+      | email           | password    |
+      | maran2@test.com | 12345678    |
+
+    And We have the following recipes:
+      | title       	|
+      | Meatball-Stew |
+
+      @javascript
+    Scenario: A user can see when someone comments on a recipe
```
Comment (magnus-thor) [inline features/user_gets_notification_when_new_comments_are_added.feature]: fix the indentation

File: app/models/recipe.rb [hunk 4]
```diff
@@ -6,10 +6,10 @@ class Recipe < ApplicationRecord
...
+  # after_create :notify
 
-  def notify 
-    ActionCable.server.broadcast 'notifications', 
-                                  message: "<p>#{self.title} was created!</p>"
-  end
+  # def notify
+  #   ActionCable.server.broadcast 'notifications',
+  #                                 message: "<p>#{self.title} was created!</p>"
+  # end
```
Comment (magnus-thor) [inline app/models/recipe.rb]: uncomment or delete commented out lines

**Review by magnus-thor — CHANGES_REQUESTED** [review]

**Review by magnus-thor — DISMISSED** [review]

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**